src/basic_recipes/hvlines.jl

```diff
@@ -83,19 +86,13 @@ function Makie.plot!(p::Union{HLines, VLines})
 end
 
 function data_limits(p::HLines)
-    scene = parent_scene(p)
-    limits = projview_to_2d_limits(scene.camera.projectionview[])
-    itf = inverse_transform(p.transformation.transform_func[])
-    xmin, xmax = apply_transform.(itf[1], first.(extrema(limits)))
     ymin, ymax = extrema(p[1][])
-    return Rect3f(Point3f(xmin, ymin, 0), Vec3f(xmax - xmin, ymax - ymin, 0))
+    return Rect3d(Point3d(NaN, ymin, 0), Vec3d(NaN, ymax - ymin, 0))
 end
```

Comment:
Just to note - you can now ignore a dimension in limits by setting it to NaN

Reply:
Nice